e2e/example_runner.go

```diff
@@ -57,19 +57,21 @@ func (f *ExamplesFramework) Create(fileName string) {
 	// try to serialize it into Unstructured or UnstructuredList
 	data, err := ioutil.ReadFile(fileName)
 	Expect(err).NotTo(HaveOccurred())
-	jsonData, err := utilyaml.ToJSON(data)
-	Expect(err).NotTo(HaveOccurred())
-	var ust runtime.Unstructured
-	err = runtime.DecodeInto(api.Codecs.UniversalDecoder(), jsonData, &ust)
-	if err != nil {
-		var ustList runtime.UnstructuredList
-		err = runtime.DecodeInto(api.Codecs.UniversalDecoder(), jsonData, &ustList)
+
+	utils.ForEachYamlDocument(data, func(doc []byte) {
```

Comment:
Let's address this in next prs, the approach with minimal code changes is ok (and it works)